docs/api-docs/cart-and-checkout/embedded-checkout-overview.md

```diff
@@ -7,33 +7,32 @@
 - [Channels, Sites, and Routes APIs](#channels-sites-and-routes-apis)
 - [BigCommerce Checkout SDK](#bigcommerce-checkout-sdk)
 - [Logged-In customers](#logged-in-customers)
+- [FAQ](#faq)
 
 </div>
 
-Embedded Checkout lets you place BigCommerce’s checkout onto any website. Customers can check out on an external storefront while their order information syncs simultaneously to the BigCommerce Control Panel. You can see this in action within the BigCommerce for WordPress plugin, which uses the same process described here as a checkout option for merchants. View more information about the plugin in the article [BigCommerce for Wordpress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/getting-started/introduction).
-
-<a id="how-it-works"></a>
+Embedded Checkout lets you place BigCommerce's checkout onto any website. Customers can check out on an external storefront while their order information syncs simultaneously to the BigCommerce Control Panel. You can see this in action within the BigCommerce for WordPress plugin, which uses the same process described here as a checkout option for merchants. View more information about the plugin in the article [BigCommerce for Wordpress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/getting-started/introduction).
```

Comment:
Embedded Checkout lets you place BigCommerce's checkout onto any website. Customers can check out on an external storefront while their order information syncs simultaneously to the BigCommerce control panel. You can see this in action within the BigCommerce for WordPress plugin, which uses the same process described here as a checkout option for merchants. View more information about the plugin in the article [BigCommerce for Wordpress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/getting-started/introduction).